src/main/java/io/github/thebusybiscuit/slimefun4/api/items/ItemGroup.java

```diff
@@ -214,11 +216,7 @@ public void remove(@Nonnull SlimefunItem item) {
      */
     public @Nonnull ItemStack getItem(@Nonnull Player p) {
         return new CustomItemStack(item, meta -> {
-            String name = Slimefun.getLocalization().getItemGroupName(p, getKey());
-
-            if (name == null) {
-                name = item.getItemMeta().getDisplayName();
-            }
```

Comment:
how come this was changed?

Reply:
`getDisplayName` has same logic with this piece of code, so I simplify it. if u don't like it I can revert this part.